packages/cli/README.md

```diff
@@ -251,7 +251,27 @@ Run the following command to install the CLI.
       url  # URL or file path of the OpenAPI spec  Type: String  Required: false
     ```
 
-10. To list available commands
+10. To discover a model from a supported datasource
+
+    ```sh
+    cd <your-project-directory>
+    lb4 discover
+    ```
+
+    ```sh
+    Usage:
+      lb4 discover [options]
+
+    Options:
+            --dataSource     # The name of the datasource to use
+            --views          # Whether the prompt should list views                                     default: true
+            --all            # Skip the model selection prompt and discover all of them                 default: true
+            --outDir         # The directory in which the `<model>.<model>.ts files will be created     default: src/models
+            --schema         # The schema with which the datasource will discover models
+
+    ```
+
+11) To list available commands
```

Comment:
Can you change this to `11.` instead of `11)` (same for `12.`)?

Reply:
nice catch